lambda/onomondo/getAllUsedSimsOnomondo.ts

```diff
@@ -0,0 +1,43 @@
+import { fetchAndValidate } from '../fetchAndValidate.js'
+import { Type, type Static } from '@sinclair/typebox'
+import { fromEnv } from '@nordicsemiconductor/from-env'
+
+const { APIKEY_ONOMONDO } = fromEnv({
+	APIKEY_ONOMONDO: 'APIKEY_ONOMONDO',
+})(process.env)
+
+export const getAllUsedSimsOnomondo = async (): Promise<Array<string>> => {
+	const iccidArray: Array<string> = []
+	let nextPage = ''
+	let hasMore = true
+	const day = `time:${new Date().toISOString().slice(0, 10)}`.replace(
+		':',
+		'%3A',
+	)
+	while (hasMore) {
+		const apiRes = await fetchAndValidate(
+			AllSimInfo,
+			`https://api.onomondo.com/usage?${nextPage}&filter=${day}`,
```

Comment:
style: use `URLSearchParams` to construct URL parameters.

Reply:
like this? 
https://github.com/bifravst/sim-details/pull/11/commits/5688c7d05ade68d94d6935b0272f8a605ac9fe8d#diff-524f8bff55fa68ecc6e8f0d838082b1068e5949107f7b31854a578bfed680fd1